index.html

```diff
@@ -291,6 +291,42 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
+    <footer>
+      <p class="footer-text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+      <div class="contact-form-container">
+        <form action="https://formspree.io/f/mrgrjoyw" method="POST" class="contact-form">
+          <input
+            type="text"
+            class="contact-form-input"
+            name="name"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="text"
+            class="contact-form-input"
+            name="name"
+            placeholder="Email address"
+            required
+          />
```

Comment:
In order to respect the project requirements and to ensure that our form is valid and functional, we need to add the type and name of `email` in the form.

- [ ] Please make sure that you modify this input with `email` requirements for the form instead of the repeating input for `name`

Reply:
Thank you for noticing this mistake 🙏 . It's been resolved.